documentation/assemblies/upgrading/assembly-upgrade-kafka.adoc

```diff
@@ -3,16 +3,31 @@
 // assembly-upgrade.adoc
 
 [id='assembly-upgrade-kafka-{context}']
-= Strimzi and Kafka upgrades
+= Required upgrade sequence
 
-Upgrading Strimzi is a three-stage process. 
-To upgrade brokers and clients without downtime, you _must_ complete the upgrade procedures in the following order:
+To upgrade brokers and clients without downtime, you _must_ complete the Strimzi upgrade procedures in the following order:
+
+. Update existing custom resources to support the `v1beta2` API version.
++
+--
+* xref:assembly-upgrade-resources-{context}[]
+--
++
+Do this after upgrading to Strimzi 0.22.0, but before upgrading to Strimzi 0.23.0 or newer.
+For a multi-version upgrade from a version prior to version 0.22.0:
++
+.. Skip this step and continue with the following steps to upgrade to version 0.22.0.
+.. Return to this step and perform all the steps in this upgrade sequence to upgrade to 0.23.0 or newer.
```

Comment:
Do you want to use the `{ConvertAfterProductVersion}` and `{ConvertBeforeProductVersion}` here as well?

Reply:
We don't need to, as we have separate downstream files. But maybe for consistency we should.

Reply:
Changed